pkg/kubectl/kubectl_test.go

```diff
@@ -27,8 +27,9 @@ func (t *TestClient) Execute(args ...string) (string, error) {
 	return "", fmt.Errorf("Missing response for %q", cmd)
 }
 
-func (t *TestClient) ExecuteStdout(args ...string) (string, error) {
-	return Execute(t, args...)
+func (t *TestClient) ExecuteOutputMatrix(args ...string) (string, string, string, error) {
```

Comment:
`string, string, string`!

Reply:
It wasn't obvious the 3rd one was the combined output.

Reply:
is there a way to make it obvious in golang?

Reply:
named return values!